yumi_description/urdf/yumi.urdf

```diff
@@ -0,0 +1,956 @@
+<?xml version="1.0" ?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from yumi.urdf.xacro                | -->
+<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
+<!-- =================================================================================== -->
```

Comment:
I think the general idea here is, that you edit `yumi.urdf.xacro` instead of this file. And try to keep the values you entered as parameters, this makes it easier to modify them. (Just some params in the beginning of the file).